packages/core-data/src/utils/Typesense.js

```diff
@@ -100,7 +101,8 @@ const createTypesenseAdapter = (config: TypesenseConfig, options = {}) => (
     geoLocationField: 'coordinates',
     additionalSearchParameters: {
       query_by: config.query_by,
-      limit: config.limit || 250
+      limit: config.limit || 250,
+      sort_by: `_text_match:desc${config.default_sort ? `,${config.default_sort}:asc` : ''}`
```

Comment:
Does this string interpolation need to be done in both places (here and Peripleo utils)? I believe the `TypesenseConfig` that gets passed into here will contain the transformed `sort_by` value from Peripleo utils, not the `default_sort` config option.